ios/yaysornays.xcodeproj/project.pbxproj

```diff
@@ -243,7 +242,7 @@
 			isa = PBXResourcesBuildPhase;
 			buildActionMask = 2147483647;
 			files = (
-				81AB9BB82411601600AC10FF /* LaunchScreen.storyboard in Resources */,
+				9A7CA252262503C500ADD141 /* Launch Screen.storyboard in Resources */,
```

Comment:
Looks like it is, wondering why?

Reply:
That's a good question. Like stated above, I did not change these files manually so I believe this was work that was done by the xCode bundler.